spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+set -e  # abort on error
+
+if [ -z "$API_HOST" ]; then
+  echo "API_HOST unset, trying minishift"
+  if which minishift && minishift status | grep -i 'openshift:.*running'; then
+    export API_HOST="$(minishift ip)"
+    eval $(minishift oc-env --shell bash) # Ensure oc in PATH
+    oc login -u system:admin # With minishift, we know we can just do this
```

Comment:
where do we login to a non minishift cluster?
or do we assume we are already logged in?
I'd say running on the openshift master is a good assumption (API_HOST=localhost)

Reply:
for non-minishift BYO, script expects you to already have logged in.

Running on openshift master is inconvenient — you need manageiq checked out & working (at least test DB) to run this script, and script leaves updated files you'll want to git commit.
I should document how to `oc adm policy add-cluster-role-to-user cluster-admin you` for remote login.

I've never actually tested the script in BYO mode.  Let me try...